commands/filescommands/objectcommands/uploaddir.go

```diff
@@ -0,0 +1,235 @@
+package objectcommands
```

Comment:
`rack files container upload` seems like it would work with the current syntax, also.